content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -469,12 +480,12 @@ public class TracedClass {
 }
 ```
 
-### OpenTelemetry instrumentations libraries
+#### OpenTelemetry instrumentations libraries
 
 You can configure other instrumentations with
 [OpenTelemetry instrumentations libraries](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 
-## Other configurations
+## Other Spring autoconfiguration-based instrumentation solutions
```

Comment:
is this micrometer tracing, which is part of spring boot?

Reply:
OTel Spring autoconfiguration and Zipkin starter